android/app-example/src/main/java/com/badoo/ribs/example/app/AppRibCustomisations.kt

```diff
@@ -1,5 +1,27 @@
 package com.badoo.ribs.example.app
 
-import com.badoo.ribs.core.directory.ViewCustomisationDirectory
+import com.badoo.ribs.customisation.RibCustomisationDirectory
+import com.badoo.ribs.customisation.RibCustomisationDirectoryImpl
+import com.badoo.ribs.customisation.inflateOnDemand
+import com.badoo.ribs.example.R
+import com.badoo.ribs.example.rib.dialog_example.DialogExample
+import com.badoo.ribs.example.rib.foo_bar.FooBar
+import com.badoo.ribs.example.rib.lorem_ipsum.LoremIpsum
+import com.badoo.ribs.example.rib.switcher.Switcher
 
-object AppRibCustomisations : ViewCustomisationDirectory()
+object AppRibCustomisations : RibCustomisationDirectory by customisations({
```

Comment:
Maybe we also should provide example of using same Rib 2 times inside one parent with different customisations?

Reply:
This is not supported yet. I believe it's really an edge-case and would not prioritise it now.

Reply:
Ok